metadata.json

```diff
@@ -32,16 +32,12 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "7",
-        "8",
```

Comment:
sorry for the noise. after some back and forth we will keep redhat 8 and only drop redhat 7. can you please add 8 back?

Reply:
done